docs/src/content/docs/es/components/code.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Código
+description: Aprende a mostrar código con resaltado de sintaxis en Starlight sin bloques de código de Markdown.
+---
+
+import { Code } from '@astrojs/starlight/components';
+
+El componente `<Code>` muestra código con resaltado de sintaxis.
+Es útil cuando no es posible usar un [bloque de código de Markdown](/es/guides/authoring-content/#bloques-de-código), por ejemplo, para mostrar datos provenientes de fuentes externas como archivos, bases de datos o APIs.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Code
+	slot="preview"
+	code={`## Welcome
+
+Hello from **space**!`}
+lang="md"
+title="example.md"
+ins={3}
+/>
+
+</Preview>
+
+## Importación
+
+```tsx
+import { Code } from '@astrojs/starlight/components';
+```
+
+## Uso
+
+Usa el componente `<Code>` para renderizar código con resaltado de sintaxis, por ejemplo, al mostrar código obtenido de fuentes externas.
+
+Ver la [documentación de Expressive Code “Componente de código”](https://expressive-code.com/key-features/code-component/) para obtener detalles completos sobre cómo usar el componente `<Code>` y la lista de props disponibles.
+
+<Preview>
+
+```mdx
+import { Code } from '@astrojs/starlight/components';
+
+export const exampleCode = `console.log('This could come from a file or CMS!');`;
+export const fileName = 'example.js';
+export const highlights = ['file', 'CMS'];
```

Comment:
And I assume the `highlights` may need to be updated based on the `exampleCode` changes.